sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/jms/properties/AzureServiceBusJmsProperties.java

```diff
@@ -128,8 +127,8 @@ public PrefetchPolicy getPrefetchPolicy() {
      *
      * @throws IllegalArgumentException If connectionString is empty.
      */
-    @PostConstruct
-    public void validate() {
+    @Override
+    public void afterPropertiesSet() throws Exception {
```

Comment:
we can do better about the connection string

Reply:
ok, I will take a close look at it.